src/app/components/Icons/Arrow.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+type ArrowProps = {
+  color: string;
+  onClick?: () => void;
+  height?: string;
+  width?: string;
+  rotate?: number;
+  draw?: boolean;
+};
+
+export default function Arrow({
+  color,
+  onClick,
+  height,
+  width,
+  rotate,
+  draw,
+}: ArrowProps): JSX.Element {
+  return (
+    <svg
+      width={width ? width : '22'}
+      height={height ? height : '22'}
+      viewBox="0 0 22 22"
+      fill={!draw ? color : 'orange'}
+      xmlns="http://www.w3.org/2000/svg"
+      onClick={onClick}
+      transform={`rotate(${!draw ? rotate : 90})`}
```

Comment:
negative conditions are harder to read
```suggestion
      fill={draw ? 'orange' : color}
      xmlns="http://www.w3.org/2000/svg"
      onClick={onClick}
      transform={`rotate(${draw ? 90 : rotate})`}
```